src/main/docs/guide/appendix/breaks.adoc

```diff
@@ -186,9 +186,6 @@ The following table summarizes the core Micronaut annotations and which are inhe
 |ann:scheduling.annotation.ExecuteOn[]                                | ❌
 |ann:scheduling.annotation.Scheduled[]                                | ❌
 |ann:session.annotation.SessionValue[]                                | ✅
-|ann:tracing.annotation.ContinueSpan[]                                | ✅
```

Comment:
this annotations are now in the tracing module: 

https://micronaut-projects.github.io/micronaut-tracing/latest/api/io/micronaut/tracing/annotation/package-summary.html

Maybe we should link them to their new home instead of removing them from the Asciidoc table.